### PR TITLE
t8ch fixes

### PR DESCRIPTION
**Pull request by t8ch**

### CODE REVIEW

**Comment by twiecki** [comment]
Thanks @t8ch, was this throwing an error before?

File: tests.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def test_automatic_changepoints_2():
 
 if __name__ == "__main__":
     test_manning()
-    test_manning_reduced_six_months()
+    # test_manning_reduced_six_months() # TODO: add SMC sampling
```
Comment (twiecki) [inline tests.py:75]: Why did you comment this out?

**Comment by t8ch** [comment]
Hi.
Yes, I got
```python
WARNING:root:Note that the automatic changepoint feature is experimental
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~/projects-gitreps/pm-prophet/tests.py in 
----> 1 test_manning_reduced_six_months()

~/projects-gitreps/pm-prophet/tests.py in test_manning_reduced_six_months()
      10     m.add_seasonality(seasonality=7, fourier_order=3)
      11     m.add_seasonality(seasonality=30, fourier_order=3)
----> 12     m.fit(method=Sampler.SMC, draws=10000)
      13     m.predict(60, alpha=0.2, include_history=True, plot=True)
      14     m.plot_components(intercept=False)

~/miniconda3/envs/bayes/lib/python3.8/enum.py in __getattr__(cls, name)
    339             return cls._member_map_[name]
    340         except KeyError:
--> 341             raise AttributeError(name) from None
    342 
    343     def __getitem__(cls, name):

AttributeError: SMC
```
I think the SMC Sampler is simply not implemented yet.

**Comment by twiecki** [comment]
@luke14free Are you still active on this project? Looking for help maintaining?

**Comment by luke14free** [comment]
I opened my own company and wish I had more free time to follow everything @twiecki 🙃 but @t8ch is trying to help me maintain the project. I'll look into active issues soon nevertheless

**Comment by t8ch** [comment]
@twiecki do you think it makes already/still sense to migrate the code to pymc4 and implement changes and new features there?

**Comment by twiecki** [comment]
No, not yet I think.

@luke14free How do you feel of moving this to its own org or the pymc3 org and giving access to some other people (like me) who can help maintaining?

**Comment by luke14free** [comment]
it makes 100% sense but I am not confident this library has the quality required to be in the pymc3 org. I am nevertheless happy to have you/tim take over/contribute.

**Pull request merged**